RecoMET/METFilters/python/metFilters_cff.py

```diff
@@ -95,39 +95,23 @@
 from RecoMET.METFilters.hfNoisyHitsFilter_cfi import *
 
 metFilters = cms.Sequence(
-   HBHENoiseFilterResultProducer *
-   HBHENoiseFilter *
-   primaryVertexFilter*
-#   HBHENoiseIsoFilter*
-#   HcalStripHaloFilter *
-   CSCTightHaloFilter *
-#   hcalLaserEventFilter *
-   #Various proposals for updated halo filters.
-   ##2015 proposals: 
-   #CSCTightHaloTrkMuUnvetoFilter *
-   #CSCTightHalo2015Filter *
-   ##2016 proposals
-   #globalTightHalo2016Filter*
-   #globalSuperTightHalo2016Filter*
-   EcalDeadCellTriggerPrimitiveFilter* 
-   ecalBadCalibFilter*
-#   *goodVertices * trackingFailureFilter *
-   eeBadScFilter*
-#   ecalLaserCorrFilter *
-#   trkPOGFilters
-   chargedHadronTrackResolutionFilter *
-   BadChargedCandidateFilter*
-   BadPFMuonFilter *
-   BadPFMuonDzFilter *
-   hfNoisyHitsFilter *
```

Comment:
This is still said to be "work in progress": don't you plan to compute this bit as well, and then select on it at the analysis level only depending on the updated reccommandations?